src/controllers/controller.bar.js

```diff
@@ -86,6 +86,10 @@ defaults._set('horizontalBar', {
 	}
 });
 
+function getValue(scale, dataPoint) {
+	return typeof dataPoint === 'string' ? Number(dataPoint) : scale.getRightValue(dataPoint);
```

Comment:
thank you! i feel dumb now. too early after waking up I guess...

I've updated the PR with this change